hamilton/reframe_parameterisation_infrastructure/reframe_tests/parameterisation_tests.py

```diff
@@ -55,14 +55,15 @@ def __init__(self, n_processors, test_id, iteration):
         self.num_tasks = 1
         self.num_cpus_per_task = n_processors
```

Comment:
Thanks - hadn't spotted the cpus_per_task in the reframe scripts! Will fix.

I did try a few options for the number of processors when running outside of reframe. 128 was quicker but what we can't account for is the current usage of the system: the jobs might end up queued for longer waiting for 128 CPUs to be free so the overall time could be longer. I thought 64 was a reasonable compromise.

Reply:
Sounds good. I'm impressed the code runs efficiently with 64 cores as this is a big increase. If its not roughly twice as fast as 32, you could just use 32 and put 4 jobs on a node.

Reply:
I think it probably works OK because each core just runs a single class, and there's no interaction between classes, so mesa just sends out a class at a time to each core it has. Just testing how long the full dataset takes.

Reply:
In single runs on the full set, using 64 cores took 20 minutes; using 32 took 37 minutes. So it's not far off twice as fast.